From 330ee764bea3ef9005f7b0c4517f8f2310000033 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:12:19 -0400
Subject: [ethernet] i40e/i40evf: bump version to 0.4.7 for i40e and 0.9.31 for i40evf

Message-id: <1409663591-32531-97-git-send-email-sassmann@redhat.com>
Patchwork-id: 90569
O-Subject: [RHEL7.1 PATCH 096/148] i40e/i40evf: bump version to 0.4.7 for i40e and 0.9.31 for i40evf
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Shannon Nelson <shannon.nelson@intel.com>

Bumpity and Fred Worm say it's time to change the numbers again.

Signed-off-by: Shannon Nelson <shannon.nelson@intel.com>
Change-ID: I658731d022ea23cedede4be2bfecd8b4cc68d270
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit e8607ef52642b7f4534b099b76c678b76f514a21)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index af45d0c..d415785 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -36,7 +36,7 @@ char i40evf_driver_name[] = "i40evf";
 static const char i40evf_driver_string[] =
  "Intel(R) XL710 X710 Virtual Function Network Driver";
 
-#define DRV_VERSION "0.9.29"
+#define DRV_VERSION "0.9.31"
 const char i40evf_driver_version[] = DRV_VERSION;
 static const char i40evf_copyright[] =
  "Copyright (c) 2013 - 2014 Intel Corporation.";
-- 
1.7.1